showcase/src/app/shared/shared.module.ts

```diff
@@ -23,11 +23,12 @@ import { FileService } from "./services";
 		MatTabsModule,
 		CommonModule,
 		StarkPrettyPrintModule,
-		TranslateModule
+		TranslateModule,
+		StarkSvgViewBoxModule
```

Comment:
Can you add a comment here to mention that this is needed so that the `SvgViewBox` directive works correctly in every module?